accelerators/federated-learning/automated_provisioning/bicep/AMLWorkspace.bicep

```diff
@@ -0,0 +1,90 @@
+@description('Specifies the name of the workspace.')
+param workspacename string
+@description('Specifies the location of the Azure Machine Learning workspace and dependent resources.')
+param location string = resourceGroup().location
+@description('Specifies whether to reduce telemetry collection and enable additional encryption.')
+param hbi_workspace bool = false
+var tenantId = subscription().tenantId
```

Comment:
Suggestion: add some newlines to separate parameters, variables, and resources.

E.g.,

```bicep
@description('Specifies whether to reduce telemetry collection and enable additional encryption.')
param hbi_workspace bool = false

var tenantId = subscription().tenantId

resource storageAccountName 'Microsoft.Storage/storageAccounts@2021-06-01' = {
    ...
```